docs/sources/clients/promtail/configuration.md

```diff
@@ -705,6 +706,27 @@ tenant:
   [ value: <string> ]
 ```
 
+#### replace
+
+The replace stage is a parsing stage that parses a log line using
+a regular expression and replaces the log line.
+
+```yaml
+replace:
+  # The RE2 regular expression. Each named capture group will be added to extracted.
+  # Each capture group and named capture group will be replaced with the value given in `replace`
```

Comment:
All these suggestions are for formatting.  The goals is to eliminate horizontal scroll bars in the published docs.
```suggestion
  # Each capture group and named capture group will be replaced with the value given in
  # `replace`
```